jack-and-the-beans/buddies-ios
# Bud 102 Favorite Topics from Activities Table

Little star in Activities table that lets you favorite a topic

Also fix annoying constraint problem with Activity cell.

To test:
1. Go to Topics
2. Click a topic, verify the star is blue if you've favorited it, or empty if you haven't
3. Toggle the star
4. Verify that it's toggled on the main topics tab

_NOTE: This also resolves #63_
